api/api/src/main/scala/uk/ac/wellcome/platform/api/models/Aggregation.scala

```diff
@@ -34,8 +35,12 @@ object Aggregations extends Logging {
           genres = e4sAggregations.decodeAgg[Genre[IdState.Minted]]("genres"),
           productionDates = e4sAggregations
             .decodeAgg[Period[IdState.Minted]]("productionDates"),
-          language = e4sAggregations
-            .decodeAgg[Language]("language", Some("data.language")),
+          language = e4sAggregations.decodeAgg[Language](
+            "language",
+            documentPath = Some("data.language")
+          ),
+          languages = e4sAggregations.decodeAgg[Language]("languages")(
+            decodeLanguageList),
```

Comment:
Is this a temporary (until the old `language` is removed) explicit override of the implicit decoder here? If so, would it make more sense to do this the other way round ie to override the decoder for `language` pending its removal / unambiguous implicit?

Reply:
Yes, although looking at it some more I don't think it's as necessary as I thought it was.

I hadn't looked at how we populate languages previously – turns out we're pretty strict about it, and always use the code, so maybe I can tidy this up.